include/ruby/internal/intern/proc.h

```diff
@@ -46,6 +46,7 @@ VALUE rb_method_call_with_block(int, const VALUE *, VALUE, VALUE);
 VALUE rb_method_call_with_block_kw(int, const VALUE *, VALUE, VALUE, int);
 int rb_mod_method_arity(VALUE, ID);
 int rb_obj_method_arity(VALUE, ID);
+VALUE rb_callable_receiver(VALUE);
```

Comment:
do not add public API without any discussion.
At least, there is no `rb_callable` methods in public C-API.

Reply:
Oh, sorry. I thought this was an 'internal' API - that's where the header is?

Or do you mean the symbol is literally visible after linking?

Do you want to revert the merge and let me find a way to fix it from where?

Reply:
now it is confusing.

```
RBIMPL_SYMBOL_EXPORT_BEGIN()

// functions are exposed

RBIMPL_SYMBOL_EXPORT_END()
```

maybe toplevel/internal/xxx will be a good place.